.github/dependabot.yml

```diff
@@ -0,0 +1,36 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+
+version: 2
+updates:
+  - package-ecosystem: "gradle" # See documentation for possible values
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "weekly"
+      time: "09:00"
+      timezone: "US/Pacific"
+    commit-message:
+      prefix: "[Gradle]"
+    open-pull-requests-limit: 10
+    target-branch: develop
+    labels:
+      - "gradle"
+      - "vulnerability"
```

Comment:
Did you create the label `vulnerability` and `gradle` in the repo?

Reply:
Yes :)